ai_economist/foundation/scenarios/simple_wood_and_stone/layout_from_file.py

```diff
@@ -185,24 +184,70 @@ def __init__(
             ranked_skills = np.array(
                 [
                     np.sort(
-                        np.minimum(pmsm, (pmsm - 1) * np.random.pareto(4, size=4) + 1)
+                        np.minimum(
```

Comment:
Do not nest numpy operators. Define intermediate variables that describe the input to np.array, np.sort, etc